client/my-sites/purchases/main.tsx

```diff
@@ -97,3 +98,27 @@ export function PurchaseCancel( {
 		</Main>
 	);
 }
+
+export function PurchaseCancelDomain( {
+	purchaseId,
+	siteSlug,
+}: {
+	purchaseId: number;
+	siteSlug: string;
+} ) {
+	const translate = useTranslate();
+
+	return (
+		<Main className="purchases is-wide-layout">
+			<DocumentHead title={ translate( 'Cancel domain' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'Cancel Domain' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-level-billing-cancel-domain)

Reply:
They are the same; this just changes the string to sentence-case.